### PR TITLE
Improve clarity in docs

### PR DESCRIPTION
**Pull request by noniagriconomie**
Feels wrong at first, Or i am mistaken?

### CODE REVIEW

File: Resources/doc/index.rst [hunk 1]
```diff
@@ -9,7 +9,7 @@ DoctrineMongoDBBundle
 
 The `MongoDB`_ Object Document Mapper (ODM) is much like the Doctrine2 ORM
 in its philosophy and how it works. In other words, like the `Doctrine2 ORM`_,
-with the Doctrine ODM, you deal only with plain PHP objects, which are then
+with the MongoDB ODM, you deal only with plain PHP objects, which are then
 persisted transparently to and from MongoDB.
```
Comment (greg0ire) [inline Resources/doc/index.rst:13]: I would leave like this for consistency with the preceding part of the sentence, but you do have a point, and I think the issue could be fixed like this:

```suggestion
persisted transparently to and from MongoDB in our case.
```

Comment (noniagriconomie) [reply Resources/doc/index.rst:13]: yes agreed, lets do this 👍

**Review by greg0ire — APPROVED** [review]

**Comment by malarzm** [comment]
@noniagriconomie thanks for the PR! Would you mind squashing commits into one please?

**Comment by noniagriconomie** [comment]
I edit it via github gui, need to get it on my laptop but i am afk right now :/
Will do it

**Comment by greg0ire** [comment]
@noniagriconomie don't bother, I did it for you :)

**Pull request merged**